src/sass/style.scss

```diff
@@ -25,28 +31,34 @@
 	font-weight: bold;
 }
 
+.successPostTweet {
```

Comment:
any reason why the success/error have different right positions?  If they are the same element, use can use a shared class for shared css and then use the additional class to change the color